### PR TITLE
Fixed ``JSONField`` deprecation warnings

### PR DESCRIPTION
**Pull request by fao89**
closes #9119

### CODE REVIEW

**Review by bmbouter — APPROVED** [review]
Thanks!

**Comment by bmbouter** [comment]
@daviddavis did you think this needs anything more?

**Review by daviddavis — APPROVED** [review]

**Review by newswangerd — APPROVED** [review]

File: pulp_ansible/app/modelresource.py [hunk 1]
```diff
@@ -30,6 +30,9 @@ class CollectionVersionContentResource(BaseContentResource):
     Resource for import/export of ansible_collectionversion-content entities.
     """
 
+    contents = fields.Field(
+        column_name="contents", attribute="contents", widget=JSONWidget(), default=list()
```
Comment (daviddavis) [inline pulp_ansible/app/modelresource.py:34]: Nice find :+1:

**Review by daviddavis — APPROVED** [review]

File: pulp_ansible/app/modelresource.py (hunk 1 above)
Comment (daviddavis) [reply pulp_ansible/app/modelresource.py:34]: Bonus points if you want to add a comment here too:

```
# required for jsonb fields in PostgreSQL database
```

Comment (fao89) [reply pulp_ansible/app/modelresource.py:34]: Actually, their introspection handles it well, the problem happens when the default is list.
For the other JSONFields that had dict as default, I didn't need to do anything

**Pull request merged**